doc/features/ffmpeg/av1.md

```diff
@@ -11,57 +11,57 @@
 
 # AV1 Encoding
 
-Not supported by upstream ffmpeg yet. Required patches are available here:
+Recommended version of VPL runtime is [intel-onevpl-22.6.0](https://github.com/oneapi-src/oneVPL-intel-gpu/releases/tag/intel-onevpl-22.6.0)
+or later. Earlier versions also support AV1, but the following patches were missing:
```

Comment:
missing -> needed

Reply:
let's handle this separately in master branch